community-mesh-vpn-tunneldigger/Makefile

```diff
@@ -0,0 +1,16 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=community-mesh-vpn-tunneldigger
+PKG_VERSION:=0.0.1
+PKG_RELEASE:=1
+
+PKG_LICENSE:=GPL-2.0-or-later
+
+include $(TOPDIR)/../package/gluon.mk
+
+define Package/community-mesh-vpn-tunneldigger
+  TITLE:=Support for connecting meshes via tunneldigger/L2TPv3 pseudowire
+  DEPENDS:=+gluon-core +gluon-mesh-vpn-core +tunneldigger +simple-tc
+endef
+
+$(eval $(call BuildPackageGluon,community-mesh-vpn-tunneldigger))
```

Comment:
You can use the $(PKG_NAME) variable instead to make it easier, when renaming this package if wanted